sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -382,6 +382,14 @@ case class AlterNamespaceSetLocationStatement(
     namespace: Seq[String],
     location: String) extends ParsedStatement
 
+/**
+ * ALTER (DATABASE|SCHEMA|NAMESPACE) ... SET OWNER command, as parsed from SQL.
+ */
+case class AlterNamespaceSetOwnerStatement(
```

Comment:
we use statement when we need to switch between v1 and v2 commands. This is a new command and there is no legacy v1 command, we can just create a new command using v2 catalog APIs.

Reply:
I don't quite follow this catalog change, do we have an example here?

Reply:
Please see `DeleteFromTable`